spec/controllers/appointments_controller_spec.rb

```diff
@@ -63,6 +63,46 @@
       end
     end
 
+    describe "filtering by days overdue" do
+      it "displays patients only less than one year overdue when checked" do
```

Comment:
Can use contexts to capture `when checked` and `when unchecked`? Furthermore, can we then set up our test data using `let!`s outside of the example itself?

Reply:
I'd really rather not extract `let`s for situations like this...will explain further on a different reply.